ragmac_xdem/main.py

```diff
@@ -146,50 +147,180 @@ def main(case: dict, mode: str, run_name: str, sat_type: str = "ASTER", nproc: i
         
         outfig = os.path.join(outdir, pair_id+"_coreg_fig.png")
         
-        if not os.path.exists(outfig):
+        if qc:
```

Comment:
Just one thought. We should try to move this out of main. Instead have a function in plotting.py or io.py.

Reply:
Yes - agreed.